### PR TITLE
OE-604 Fix delete data wordings for OEB

### PR DESCRIPTION
**Pull request by ErnestFan**
**What's this do?**
The alert after deleting server data now shows Open eBooks instead of SimplyE
Bump version to 2.5.1

**Why are we doing this? (w/ JIRA link if applicable)**
[OE-604](https://jira.nypl.org/browse/OE-604)

**How should this be tested? / Do these changes have associated tests?**
Log into Open eBooks
Go to 'Account' -> 'Delete Server Data'
After deleting server data, it should show an alert with Open eBooks name on it

**Dependencies for merging? Releasing to production?**
N/A

**Does this include changes that require a new SimplyE/Open eBooks build for QA?**
Yes

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@ErnestFan

### CODE REVIEW

File: Simplified/it.lproj/Localizable.strings
Comment (ErnestFan) [inline Simplified/it.lproj/Localizable.strings:266]: @ettore Can you check if the Italian wording looks right to you?

Comment (ettore) [reply Simplified/it.lproj/Localizable.strings:266]: actually these strings do look bit odd.... can you change both OE and SE strings to say: "I dati del tuo utente di SimplyE / Open eBooks sono stati rimossi"

Comment (ettore) [reply Simplified/it.lproj/Localizable.strings:266]: also at line 260 / 261 let's just do simply for both apps: "Rimuovi i miei dati". It's implicit we're talking about SE/OE, but if I add SimplyE / OpenE it always sounds weird in Italian. (Italian doesn't have the Saxon genitive form, so I'd be forced to use the equivalent of `of`, and that gets weird because we're referring to "my" data, but also the data "of" SimplyE/OE....)

File: Simplified.xcodeproj/project.pbxproj
Comment (ettore) [inline Simplified.xcodeproj/project.pbxproj:5774]: this is now obsolete since we are already at build 3 for OE 2.5.1 on develop, I think. Which makes me think that it's also weird that GitHub reports 2.5.1 as a change, since it's already on develop.

Do you want to verify/update (or remove) the version.build numbers and just merge this PR? there are just minor changes in the Italian localization.

Comment (ErnestFan) [reply Simplified.xcodeproj/project.pbxproj]: Updated the Italian wordings and removed the version.build numbers. However, I will wait until the audiobook feature branch is merged, to avoid conflicts on the other PR.

**Review by ettore — APPROVED** [review]

**Pull request merged**
